lib/backends/sdc/containers.js

```diff
@@ -1735,6 +1953,21 @@ function buildVmPayload(opts, container, callback) {
             addNetworksToPayload(opts, container, payload, cb);
         },
 
+        function addExternalNet(_, cb) {
+            if (publishingPorts(container) || opts.fabricRequireExternal) {
+                // external must be primary.
+                payload.networks.forEach(function (nw) {
+                    delete nw.primary;
+                });
+
+                payload.networks.push(
+                    { uuid: opts.config.overlay.externalPool, primary: true });
```

Comment:
When the `sdc_nat_pool` was added in NET-270 it was marked as optional; however, I think that the way we've been doing things, it's been treated as required in sdcadm. Likely some additional follow up work should be done to not assume it'll be there.